troupon/authentication/templates/authentication/login.html

```diff
@@ -48,7 +48,7 @@
                         <div class="row" style="margin-left:155px;">
```

Comment:
+ Move the facebook login link inside this div, then remove the inline styling here.  All stylings should be in the sass/css file.

+ use `text-align: center` on this div instead of `margin-left: 155px;`